ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js

```diff
@@ -1,10 +1,261 @@
 
 [{
   zoneId: ZoneId.ThePuppetsBunker,
   timelineFile: 'the_puppets_bunker.txt',
   triggers: [
+    {
+      id: 'Puppet Aegis Anti-Personnel Laser',
+      netRegex: NetRegexes.headMarker({ id: '00C6' }),
+      condition: Conditions.targetIsYou(),
+      // TODO: do we need a second "away from tank" trigger if you don't have a marker?
+      // We don't do this in Copied Factory for this same 00C6, but we could?
+      response: Responses.tankBuster('info'),
+    },
+    {
+      id: 'Puppet Aegis Beam Cannons',
+      netRegex: NetRegexes.startsUsing({ source: '813P-Operated Aegis Unit', id: '5073', capture: false }),
+      alertText: function(data) {
+        data.beamCannonCount = data.beamCannonCount || 0;
+        data.beamCannonCount++;
+
+        // TODO: 507[789A] is probably a rotation ability sync, so we could give better directions?
+        if (data.beamCannonCount <= 2) {
+          return {
+            en: 'Go To Intersection',
+          };
+        }
+        return {
+          en: 'Go To Narrow Intersection',
+        };
+      },
+    },
+    {
+      id: 'Puppet Aegis Aerial Support Surface Laser',
+      netRegex: NetRegexes.headMarker({ id: '0017' }),
+      condition: (data, matches) => data.me === matches.target && data.phase !== 'superior',
+      // TODO: should this be more along the lines of an imperative, like GTFO?
+      alarmText: {
+        en: '10x Sky Laser on YOU',
+      },
+    },
+    {
+      id: 'Puppet Aegis Refraction Cannons 1',
+      netRegex: NetRegexes.startsUsing({ source: '813P-Operated Aegis Unit', id: '5080', capture: false }),
+      alertText: function(data) {
+        return {
+          en: 'Get Under Wings',
+        };
+      },
+    },
+    {
+      id: 'Puppet Aegis Refraction Cannons 2',
+      netRegex: NetRegexes.startsUsing({ source: '813P-Operated Aegis Unit', id: '507F', capture: false }),
+      alertText: function(data) {
+        return {
+          en: 'Get Between Wings',
+        };
+      },
+    },
+    {
+      id: 'Puppet Aegis High-Powered Laser',
+      // This is also head marker 003E, but since there's three stacks, just say "stack".
+      netRegex: NetRegexes.startsUsing({ source: '813P-Operated Aegis Unit', id: '508F', capture: false }),
+      response: Responses.stack(),
+    },
+    {
+      id: 'Puppet Aegis Life\'s Last Song',
+      netRegex: NetRegexes.startsUsing({ source: '813P-Operated Aegis Unit', id: '53B3', capture: false }),
+      // This is more a "if you haven't done this ever or in a while, here's a reminder."
+      // Tell it once, but as this repeats nearly continously forever, only say it once.
+      suppressSeconds: 9999,
+      infoText: {
+        en: 'Dodge into ring gap',
+      },
+    },
+    {
+      id: 'Puppet Light Volt Array',
+      netRegex: NetRegexes.startsUsing({ source: 'Light Artillery Unit', id: '5211' }),
+      condition: (data) => data.CanSilence(),
```

Comment:
As a general rule, are we moving to a preference for arrow functions in conditions? I have no strong preference either way, but I can definitely see the appeal of saving two lines.

Reply:
I think when it's a one-liner, I have been.  I was thinking once 5.3 stuff was done, I'd replace `function` with arrow functions in all triggers, if that makes sense?